src/scss/initial/_root.scss

```diff
@@ -3,6 +3,9 @@
 
 html {
   @include config.font-family('body');
+  @include config.ratios--;
+  @include config.times--;
+  @include config.changes--;
```

Comment:
I didn't think that ratios, times and changes would belong in their own initial/* partials like the scale and colors partials but I am used to seeing all these included at once.